descartes_core/CMakeLists.txt

```diff
@@ -3,13 +3,12 @@ project(descartes_core)
 
 add_definitions(-std=gnu++0x)
 find_package(catkin REQUIRED COMPONENTS
-  console_bridge
   moveit_core
   roscpp
+  cmake_modules
 )
 
 find_package(Boost REQUIRED)
-find_package(Eigen REQUIRED)
```

Comment:
Why is this removed if `Eigen` is listed as a `DEPENDS` in `catkin_package(..)` further on?

Also: if you don't use `Eigen` in this package, then it would seem there is also no need for `cmake_modules`?